src/api.c

```diff
@@ -536,10 +536,16 @@ STATIC int cgroup_parse_rules_options(char *options, struct cgroup_rule * const
 		cmp_len = min(strlen(stok_buff), strlen(CGRULE_OPTION_IGNORE));
 		if (strlen(stok_buff) == strlen(CGRULE_OPTION_IGNORE) &&
 		    strncmp(stok_buff, CGRULE_OPTION_IGNORE, cmp_len) == 0) {
-			rule->is_ignore = true;
+			rule->is_ignore = 1;
 			continue;
 		}
 
+		cmp_len = min(strlen(stok_buff), strlen(CGRULE_OPTION_IGNORE_RT));
+		if (strlen(stok_buff) == strlen(CGRULE_OPTION_IGNORE_RT) &&
+		    strncmp(stok_buff, CGRULE_OPTION_IGNORE_RT, cmp_len) == 0) {
+			rule->is_ignore = 2;
```

Comment:
The way this is currently written, we would not be able to handle `ignore,ignore_rt` in the options field.  Is that a valid configuration?

Reply:
If it's invalid, we should probably error out if someone has tried to set both.  If it's valid, then maybe a bit field?

Reply:
@drakenclimber Thank you so much for the review. Agreed, I will fix the code for all three cases:
1. Only `ignore` option
2. Only `ignore_rt` option
3. Both `ignore` and `ignore_rt` options.

Sure, good idea will implement something on the same lines.